### PR TITLE
Discord embed now shows the correct word if the last guess is wrong.

### LINKED ISSUE #4: Upon losing, show what the word of the game was

### PR DESCRIPTION
**Pull request by davemcwave**
There is now a text description at the top of the embedded image, telling you the correct word. But this only shows up after the last incorrect word guess.
image

Of course, the correct word guess still works as normal.
image

### CODE REVIEW

**Comment by davemcwave** [comment]
This addresses https://github.com/ChristianLowe/Discordle/issues/4.

File: src/main/java/io/chrislowe/discordle/game/GameManager.java [hunk 1]
```diff
@@ -32,6 +32,10 @@ public void startNewGame() {
         gameIsActive = true;
     }
 
+    public Game getGame() {
```
Comment (HelloChrisLowe) [inline src/main/java/io/chrislowe/discordle/game/GameManager.java]: If practical, I would prefer if we can keep the encapsulation of the Game object inside of GameManager. Do you think we can write this similarly to #getWordGuesses which pulls directly from the internally managed game object?

File: src/main/java/io/chrislowe/discordle/Main.java [hunk 2]
```diff
@@ -112,23 +112,33 @@ private static void registerCommands(GatewayDiscordClient gateway) {
 
                 if (response != null) {
                     yield event.reply(response);
-                } else {
-                    yield event.deferReply().then(createGameBoardFollowup(event));
+                }
+                else {
```
Comment (HelloChrisLowe) [inline src/main/java/io/chrislowe/discordle/Main.java]: This is very minor, but this project is following standard Oracle styling guidelines, which have an `else` statement on the same line as the previous bracket, like this:
`} else {`

**Review by HelloChrisLowe — CHANGES_REQUESTED** [review]

File: src/main/java/io/chrislowe/discordle/game/GameManager.java (hunk 1 above)
Comment (davemcwave) [reply src/main/java/io/chrislowe/discordle/game/GameManager.java]: Done!

File: src/main/java/io/chrislowe/discordle/Main.java (hunk 2 above)
Comment (davemcwave) [reply src/main/java/io/chrislowe/discordle/Main.java]: Done!

**Comment by HelloChrisLowe** [comment]
Thank you for the PR!

**Pull request merged**